browser-bunyan/browser-bunyan-tests.ts

```diff
@@ -0,0 +1,7 @@
+var bunyan = require('browser-bunyan');
```

Comment:
Needs TypeScript import syntax (`import * as bunyan from "browser-bunyan"`) or this file won't be typechecked.